packages/graalvm/src/main/kotlin/elide/runtime/intrinsics/js/stream/WritableStreamSink.kt

```diff
@@ -0,0 +1,12 @@
+package elide.runtime.intrinsics.js.stream
```

Comment:
plz run `spotlessApply` or add copyright headers. we should figure out why those aren't automating

Reply:
Yeah I'll run it before marking as ready, it's easier to do it all in one pass.